.github/actions/maven-build-and-tag/action.yml

```diff
@@ -190,6 +197,8 @@ runs:
         MAVEN_CLI_OPTS: ${{ steps.compute-maven-options.outputs.result }} -Dlogging.root.level=off -Dspring.main.banner-mode=off -Ddocker.skip
         MAVEN_USERNAME: ${{ inputs.maven-username }}
         MAVEN_PASSWORD: ${{ inputs.maven-password }}
+        GHPKG_USERNAME: ${{ inputs.ghpkg-username }}
+        GHPKG_PASSWORD: ${{ inputs.ghpkg-password }}
```

Comment:
Sorry but you just need those in the env context when the maven command runs, right?
Not sure you need those inputs: putting the same in the env when calling this action should be enough no?